packages/serverpod_client/lib/src/serverpod_client_shared.dart

```diff
@@ -153,7 +153,6 @@ abstract class ServerpodClientShared extends EndpointCaller {
   }
 
   /// Closes all open connections to the server.
-  @override
```

Comment:
Is this change required within the scope of this PR?

Reply:
Did I make these pushes in the same branch? Oops I need to revert them. Sorry. This is invalid override.

Reply:
Did I make these pushes in the same branch? Oops I need to revert them. Sorry. This is invalid override.